main.js

```diff
@@ -789,7 +786,14 @@ function sendRange(req, res, buffer) {
 //    "bind":   "0.0.0.0", // "::"
 //    "cache":  false
 //}
-function initWebServer(settings) {
+async function initWebServer(settings) {
+
+    if (adapter.config.secure) {
+        // Load certificates and/or get Lets Encrypt config.
+        const certObj = await adapter.getCertificatesAsync();
+        adapter.config.certificates = certObj[0];
+        adapter.config.leConfig = certObj[1];
+    }
```

Comment:
In this section, you don't use `settings`, but `adapter.config` (which is what was given to this method in the `settings` parameter). I suggest you either use `adapter.config` everywhere (and get rid of the `settings` parameter) or you use `settings` instead of `adapter.config` in this entire section (and further below).

Reply:
Fair point. Updated. https://github.com/raintonr/ioBroker.web/commit/f16c7cb2fde89968d0ada7044835debcacbc0109